dscheck.opam

```diff
@@ -11,6 +11,7 @@ depends: [
   "dune" {>= "2.9"}
   "containers"
   "oseq"
+  "alcotest" {>= "1.6.0"}
```

Comment:
Drive-by-comment: :sweat_smile: 
I think this should be a `with-test` dependency

Reply:
haha, thanks!